### PR TITLE
fix: fix weird & invalid md format

### PR DESCRIPTION
**Pull request by littlecxm**
fix: fix weird & invalid md format

### CODE REVIEW

File: docs/Community_IRC.md [hunk 1]
```diff
@@ -1,100 +1,91 @@
...
 Everybody is free to join any of these channels.
 We may or may not add more channels in future depending on the needs.
-<br/><br/>
 
-## 👮 --------------- Services
+## 👮 Services
 
 Besides the services offered by Libera (like Nickserv or Chanserv) Armbian has set up some own services.  
 
-- **ArmbianGithub**
```
Comment (EvilOlaf) [inline docs/Community_IRC.md:54]: This is the actual name of the bot.

**Review by EvilOlaf — CHANGES_REQUESTED** [review]

File: docs/Community_IRC.md [hunk 2]
```diff
@@ -1,100 +1,91 @@
...
+- **Armbian Github**
     - Has the purpose to fill #armbian-commits and #armbian-rss like described above
-- **Armbian-Discord**
```
[the remark this answers is not in the document]
Comment (littlecxm) [reply docs/Community_IRC.md:56]: sorry change it back

**Comment by igorpecovnik** [comment]
> okay and i suggest the better way is publish binary pdf to release through CI

Yes. This would be nice to refactor some day.

**Comment by littlecxm** [comment]
> > okay and i suggest the better way is publish binary pdf to release through CI
> 
> Yes. This would be nice to refactor some day. https://armbian.atlassian.net/browse/AR-1147

https://github.com/armbian/documentation/pull/199

**Pull request merged**